src/utils/localSync.ts

```diff
@@ -317,9 +320,41 @@ const findNodeAfterVerse = (
   return after;
 };
 
-const postPass = (doc: Document, p: Passage, memory: Memory) => {
+const ParseTranscription = (currentPassage: Passage, transcription: string) => {
+  var pattern = /(\\v\s*[1-9+]-*[1-9+]*)/g;
+  var internalverses = Array.from(transcription.matchAll(pattern));
+  // Get all matches
+  if (internalverses.length <= 1) {
+    currentPassage.attributes.lastComment = transcription.trimEnd();
```

Comment:
So I am not sure why we set the comment value to the transcription value?

Reply:
because I'm setting the comment to the chunk of the transcription that corresponds to this passage...i.e. if we come in with 1-5 but the transcription contains \v1 words for 1 \v2 words for 2, I create two passage records 1 and 2 and set their comment to their part of the transcription.  Then I process the list of passages as if that's what came in.  If it's too hacky, I could return an array of {passage,transcription} but the comment was there, so I used it.  In this case, the transcription didn't contain \v so I'm using it as is, but the loop assumes the transcription is in the comment.